IQSS/dataverse
# Make geospatial coverage "Other" field facetable

**What this PR does / why we need it**:
This PR lets Dataverse admins add a search facet on their collection pages for the Geospatial metadatablock's "Other" field, so that others can narrow searches in their collections using the values entered in that "Other" field.

**Which issue(s) this PR closes**:

Closes #7399

**Special notes for your reviewer**:

**Suggestions on how to test this**:

- Edit a Dataverse collection (Edit > General Information) to enable the Geospatial metadatablock (in the Metadata Fields section) and in the in the Browse/Search Facets section, add the Geospatial metadatablock's "Other" field as a facet.
- Create a dataset with a value entered in the Geospatial metadatablock's "Other" field, save the dataset, and check if the Dataverse collection's page has an "Other" facet with a link for the value you entered in the Other field.
- Click on that facet link and the page should show the dataset you created.

**Does this PR introduce a user interface change? If mockups are available, please link/include them here**:

Yes, this PR introduces a UI change. Mockups not available.

**Is there a release notes update needed for this change?**:

Yes.

**Additional documentation**:
